tests/recipes/pkg_b/conanfile.py

```diff
@@ -15,7 +15,5 @@ def package_id(self):
     self.info.header_only()
 
   def package_info(self):
-    self.cpp_info.name = "pkg_b"
-    self.cpp_info.components["pkg_b_lib"].name = "pkg_b_lib"
-    self.cpp_info.components["pkg_b_lib"].includes = ["include"]
-    self.cpp_info.components["pkg_b_lib"].requires = ["pkg_a::pkg_a_lib"]
+    self.cpp_info.includedirs = ["include"]
+    self.cpp_info.requires = ["pkg_a::pkg_a"]
```

Comment:
```suggestion
    pass
```
requires is no longer necessary, not using components.